jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/TransformRunnable.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.ideMaven
+
+class TransformRunnable : Runnable {
+    private var isComplete: Int? = null
```

Comment:
enum?

Reply:
We are trying to get the actual exitCode and emit it to the telemetry once the maven command fails. So `Int` is expected here.